### PR TITLE
fix: Use the right ssl query param for jdbcUrl

### PR DESCRIPTION
**Pull request by FelisiaM**
The correct query param to use ssl in a jdbc url is `ssl`. Replaced the old `useSSL` (which was not being used) with `ssl` query param. https://jdbc.postgresql.org/documentation/80/connect.html
By default, RDS for PostgreSQL uses and expects all clients to connect using SSL/TLS, therefore the value of ssl is set to true always. Can use `require_ssl` to enforce only accepting SSL connections.

[#183032399](https://www.pivotaltracker.com/story/show/183032399)

### Checklist:

* [x] Have you added Draft Release Notes in `docs/draft-release-notes.md`?
* [x] Have you followed the [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary)?

### CODE REVIEW

File: terraform/postgresql/bind/outputs.tf [hunk 1]
```diff
@@ -31,13 +31,12 @@ output "uri" {
...
 output "jdbcUrl" {
   value = format(
-    "jdbc:postgresql://%s:%d/%s?user=%s\u0026password=%s\u0026useSSL=%v",
+    "jdbc:postgresql://%s:%d/%s?user=%s\u0026password=%s\u0026ssl=true",
     var.hostname,
     local.port,
     var.db_name,
     csbpg_binding_user.new_user.username,
     csbpg_binding_user.new_user.password,
-    var.require_ssl,
```
Comment (jhvhs) [inline terraform/postgresql/bind/outputs.tf:40]: **question** Should we also remove the `require_ssl` parameter from the definitions?

Comment (blgm) [reply terraform/postgresql/bind/outputs.tf:40]: Agreed. This seems to be the only usage.

**Review by jhvhs — APPROVED** [review]

File: terraform/postgresql/bind/outputs.tf (hunk 1 above)
Comment (jhvhs) [reply terraform/postgresql/bind/outputs.tf:40]: My bad, it's also used to set the `rds.force_ssl` parameter

Comment (blgm) [reply terraform/postgresql/bind/outputs.tf:40]: Yes, but that’s in the provision isn’t it?

File: docs/draft-release-notes.md [hunk 2]
```diff
@@ -27,10 +27,11 @@
...
 - PostgreSQL: There are no default plans defined. Plans must be configured through the environment variable: `GSB_SERVICE_CSB_AWS_POSTGRESQL_PLANS`.
-- Terraform upgrade (from 0.12.30 to 1.1.9) has been added
+- Terraform upgrade (from 0.12.30 to 1.1.9) has been added.
 
 ### Fix:
 - minimum constraints on MySQL and PostgreSQL storage_gb are now enforced
 - adds lifecycle.prevent_destroy to all data services to provide extra layer of protection against data loss
 - Modification of the region generates the same service without eliminating the existing one in the newly established region. Blocking updating operation of such property to avoid the generation of infrastructure unintentionally.
 - PostgreSQL role is now always cleanly deleted during unbind
+- PostgreSQL jdbc URL includes the valid query parameter to use ssl. The parameter is not always set to true since RDS for PostgreSQL uses and expects all clients to connect using SSL.
```
Comment (jhvhs) [inline docs/draft-release-notes.md:37]: **Suggestion** The phrase should probably say: "The parameter is now always set to true..."

**Pull request merged**